src/extension_utils/utils.ts

```diff
@@ -372,9 +418,194 @@ export const installPythonDependencies = async (
                 }
             });
 
+        return pythonExecutable;
+    }
+
+    return pythonPath;
+};
+
+export const checkForDependencies = async (
+    context: vscode.ExtensionContext,
+    pythonPath: string
+) => {
+    const dependencyCheckerPath: string = getPathToScript(
+        context,
+        CONSTANTS.FILESYSTEM.OUTPUT_DIRECTORY,
+        HELPER_FILES.CHECK_PYTHON_DEPENDENCIES
+    );
+    try {
+        const { stdout } = await executePythonCommand(
+            pythonPath,
+            `"${dependencyCheckerPath}"`
+        );
+        console.info(stdout);
```

Comment:
As I mentioned above, I think it'd be nice to have a bit more of documentation for these -- just so it's easier to debug in the future. Like explaining why we need to output these for stdout and things like that.

Reply:
ah okok! I'll try to add more!